slynk-stepper.lisp

```diff
@@ -223,6 +223,9 @@
                          debugp)
   "Return plists representing forms of interest inside STRING.
 If DEBUGP return information about the actual forms."
+  #+armedbear
+  (format *error-output* "~&; ABCL is unsupported.~%")
```

Comment:
I would just `cl:error` here.  The user gets presented with a debugger, knows what's up and won't try it again.

Reply:
Done!